### DISCUSSION TITLE
ERP-система для призводства кормов для животных

### DISCUSSION THREAD

**Discussion by kseonyka** [totumonline/totum-ru-issues-and-discussions#26] [Сделанные проекты]
ERP-системы — это программы для управления ресурсами производства. Как правило, даже если предприниматель или управленец в курсе, что такой инструмент автоматизации существует, он не всегда готов внедрять его в своей компании — сам бизнес еще не «созрел».

ERP-система актуальна, когда бизнес-процессы отлажены хотя бы в каком-то формате, пускай даже с помощью простейших инструментов: на досках и стикерах или в Excel-таблицах. Главное — понимание, как устроено производство и где у него узкие места.

Кейс, о котором мы пишем [в этой статье](https://vc.ru/services/302455-my-uvolili-snabzhenca-posle-vnedreniya-erp-sistemy-kak-avtomatizaciya-biznes-processov-izmenila-rabotu-proizvodstva), получился удачным, потому что у клиента уже был сформированный опыт управления производством. Он на несколько шагов вперед знал, с какими проблемами сталкиваются сотрудники на производстве и в каких процессах им нужно «помочь» автоматизацией.

Мы внедряем решения, которые будут удобны для отдельной компании. Да, все функции, нужные «БЭКС», можно реализовать в 1С, но мы остановились на Totum и не ошиблись. 

_______________________________________________________________________

Работать без автоматизации — это как ехать за рулем без приборов и навигатора. Водитель движется к какой-то точке, но вести машину по правильному пути и без аварий вряд ли получится. А с оборудованием видна скорость, самая удобная дорога, объем топлива. Так и ERP-система показывает состояние дел в компании, открывает ее слабые места и помогает усилить их.

Например, программа показывает, что на производстве потери. Значит, управленцу нужно глубже «нырнуть» в этот процесс и понять, что идет не так. Может, оборудование обветшало и стало выпускать брак, а может, новый сотрудник плохо управляется с установкой и ему требуется обучение.

Или не выполняется план продаж. Нанять еще одного менеджера, обновить систему мотивации или составить более удачный скрипт общения с клиентами? Разбираться и принимать решение собственнику и топ-менеджерам, но ERP-система — тот инструмент, который подсветил проблему и заставил задуматься, что здесь не так.

_______________________________________________________________________

Благодарим команду Totum за совместный проект и отдельная благодарность Алексею за грамотное сопровождение и экспертный консалтинг в вопросах адаптации логики производственных процессов в системе 🤝

**Comment by ttm-alexeypolunin** [comment]
Спасибо за отзыв и статья хорошая получилась! Я думаю, что следующий проект у вас получится более объемным.
